### ISSUE TITLE
Cannot make a part stackable

### ISSUE THREAD

**Issue by ihsoft** [ihsoft/KIS#97]
I tried to override stackable state for the stock batteries and found out interesting behavior: for Z-400 override doesn't work in spite of it works just fine for the other batteries. Those parts have no module so, I've defined override by the name:

```
    StackableItemOverride
    {
        partName = addWhatYouWantHere1
        partName = addWhatYouWantHere2
// Batteries
        partName = batteryPack
        partName = batteryBank
        partName = batteryBankMini
        partName = ksp_r_largeBatteryPack  // <== Z-400, doesn't work
        partName = batteryBankLarge
    }
```

The only difference between Z-400 and other battery names is underscore in the name. Can it be the issue?

**Comment by ihsoft** [comment]
I've figure it out. In case of battery packs the real name is a namespace path. I.e. for `ksp_r_largeBatteryPack`  it's actually `ksp.r.largeBatteryPack`. When specified like this everything is working fine. It makes sense to add a comment in `settings.cfg`. Or translating dots in the part name into underscores when checking in the plugin.

**Comment by ihsoft** [comment]
Released in v1.2.4
